Tasks/AzureResourceGroupDeploymentV2/operations/ResourceGroup.ts

```diff
@@ -170,8 +170,29 @@ export class ResourceGroup {
             tl.error(error.message);
             if (error.details) {
                 tl.error(tl.loc("Details"));
+                var isPolicyVoilated = false;
                 for (var i = 0; i < error.details.length; i++) {
-                    var errorMessage = util.format("%s: %s %s", error.details[i].code, error.details[i].message, error.details[i].details);
+                    var errorMessage = null;
+                    if(error.details[i].code === "RequestDisallowedByPolicy")
+                    {
+                        if(!isPolicyVoilated)
+                        {
+                            /**
+                             * Get portal url and management group name from policy id in error
+                             */
+                            var managementGroupId = this.getmanagementGroupId(error.details[i]);
+                            let policyLink = "https://ms.portal.azure.com/#blade/Microsoft_Azure_Policy/PolicyMenuBlade/Assignments/scope/%2Fproviders%2FMicrosoft.Management%2FmanagementGroups%2F" + managementGroupId;
+                            tl.warning("Manage Azure policy : " + policyLink);
```

Comment:
Localize string.

Reply:
Localize all strings which are shown to the user